feature/PER-220

JosephJindrich opened the pull request:
Added 404 page to uninitialized scenes and captions.

JosephJindrich committed:
[PER-220] Added 404 page to uninitialized scenes and captions.

Markobozic reviewed (commented):
This is not performing what's expected in the ticket PER-220. navigating to unknown scenes is not producing 404's

evanhackett committed:
fix 404 route bug.

fixed bug where a caption route would break if the scene was undefined.
Also changed some if statements to match other if statement style.
